### DISCUSSION TITLE
New workspace following color_scheme

### DISCUSSION THREAD

**Discussion by RoXuS** [wezterm/wezterm#6307] [General]
Not sure if it is a bug or my incompetence, I use this shortcut to change/load new workspace:
```
    {
      key = 'l',
      mods = 'CTRL|SHIFT',
      action = wezterm.action.ShowLauncherArgs {
        flags = 'FUZZY|WORKSPACES',
      },
    },
```

And this function to load my theme (catppuccin) following system theme:
```
local function scheme_for_appearance()
  local appearance = wezterm.gui.get_appearance()
  if appearance:find "Dark" then
    return "Catppuccin Frappe"
  else
    return "Catppuccin Latte"
  end
end

...
color_scheme = scheme_for_appearance(),
...
```

My issue is when I create a new workspace the `scheme_for_appearance` function seems to be not called and the new pane is stuck to Latte. But if I create a new pane, it takes the good color scheme.

**Comment by RoXuS** [comment]
Ok after relaunch WezTerm is ok ...

**Comment by dderg** [comment]
it happens to me too, I even installed nightly build, seems like it breaks after I edit the config file.
